plugins/newrelic-dashboard/src/api/NewRelicDashboardApi.ts

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2021 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { createApiRef } from '@backstage/core-plugin-api';
+import { DashboardEntity } from '../types/DashboardEntity';
+import { DashboardSnapshot } from '../types/DashboardSnapshot';
+import { SnapshotDetails } from '../types/SnapshotDetails';
+
+export interface DashboardEntitySummary {
+  getDashboardEntity: DashboardEntity;
```

Comment:
I have been using this system to organize interfaces and get methods. It helps in grouping and makes the code more readable. This is not an ideal example, but if we need to add a few more services to the plugin then this can be useful. If you don't mind, I am Keeping it like this for now, as it won't really impact the performance nor violate the coding practices